src/org/usfirst/frc/team467/robot/XBoxJoystick467.java

```diff
@@ -85,8 +89,8 @@ public boolean buttonReleased() {
 		leftY(1),
 		leftTrigger(2),
 		rightTrigger(3),
-		rightX(4),
-		rightY(5);
+		RightX(4),
+		RightY(5);

```

Comment:
Case fixes are always good.

Reply:
Shouldn't these stay as rightX and rightY instead of RightX and RightY? Why did they change? It's not consistent with the left variables.